presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -333,6 +333,17 @@ protected void assertCreateTableAsSelect(Session session, String table, @Languag
         assertFalse(getQueryRunner().tableExists(session, table));
     }
 
+    @Test(expectedExceptions = RuntimeException.class, expectedExceptionsMessageRegExp = "Regexp matching interrupted",
+            timeOut = 30_000)
```

Comment:
put into the same line; or if it's too long, keep the first line empty and then one param per line